go/binlog/gomysql_reader.go

```diff
@@ -78,7 +84,7 @@ func (this *GoMySQLReader) GetCurrentBinlogCoordinates() *mysql.BinlogCoordinate
 
 // StreamEvents
 func (this *GoMySQLReader) handleRowsEvent(ev *replication.BinlogEvent, rowsEvent *replication.RowsEvent, entriesChannel chan<- *BinlogEntry) error {
-	if this.currentCoordinates.SmallerThanOrEquals(&this.LastAppliedRowsEventHint) {
+	if !this.migrationContext.UseGTIDs && this.currentCoordinates.SmallerThanOrEquals(&this.LastAppliedRowsEventHint) {
```

Comment:
Is there a chance that we could also receive a GTID event with an ID that has already been processed?  Or is that handled completely by the `binlogSyncer.StartSyncGTID` method?